packages/www/components/DocsLayout/guides.tsx

```diff
@@ -11,12 +11,20 @@ const guidesTree: Tree[] = [
   ],
   [
     {
-      content: "Feature Support Matrix",
+      content: "Supported codecs and workflows",
```

Comment:
Could it be something like "Software and Features Supported"?

FWIW we talk about "Software and Features Livepeer.com Supports" on the blog: https://livepeer.com/blog/integrating-with-livepeer-streaming-pipeline-software-suggestions.